qdax/core/emitters/multi_emitter.py

```diff
@@ -0,0 +1,87 @@
+from typing import Optional, Tuple
+
+import jax
+from jax import numpy as jnp
+
+from qdax.core.containers.repertoire import Repertoire
+from qdax.core.emitters.emitter import Emitter, EmitterState
+from qdax.types import Descriptor, ExtraScores, Fitness, Genotype, RNGKey
+
+
+class MultiEmitterState(EmitterState):
+    emitters_state_tuple: Tuple[EmitterState, ...]
```

Comment:
Just about the naming. What do you think about emitter_states? I am trying to get rid of the "_tuple" suffix haha

Reply:
agreed! I changed that!